x-pack/plugins/security/server/authentication/authentication_service.ts

```diff
@@ -297,7 +302,7 @@ export class AuthenticationService {
       } else if (authenticationResult.redirected()) {
         this.logger.error('Re-authentication failed since redirect is required.');
       } else {
-        this.logger.error('Re-authentication cannot be handled.');
+        this.logger.debug('Re-authentication cannot be handled.');
```

Comment:
It's actually the other way around. We might receive a lot of these log messages for fake requests and requests with HTTP authentication (e.g., when API keys have expired or have been invalidated). Such requests cannot be re-authenticated/handled (hence we return `AuthenticationResult.notHandled()`) by design since they don't have an associated session. Eventually, as we integrate more with the Core, we won't even attempt to re-authenticate.

Does that make sense?

Reply:
That makes sense. Curious about what fake requests we have coming through, but that is probably a topic to discuss elsewhere. Thanks!

Reply:
> Curious about what fake requests we have coming through, but that is probably a topic to discuss elsewhere.

Yeah, that's a good question. I've seen these in our Overview Cluster, but it's hard to know where they are coming from.